config/admin/headers.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/** Content-Security-Policy headers */
+header("Content-Security-Policy-Report-Only: "
+    ."default-src 'self'; "
+    ."style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://cdnjs.cloudflare.com https://*.bootstrapcdn.com; "
+    ."script-src 'self' 'unsafe-inline' 'unsafe-eval' https://cdnjs.cloudflare.com; "
+    ."img-src 'self' data: https://cdnjs.cloudflare.com; "
+    ."font-src 'self' data: https://*.bootstrapcdn.com https://fonts.gstatic.com; "
+    ."form-action 'self'; "
+    ."upgrade-insecure-requests; "
+    ."block-all-mixed-content; "
+    ."report-uri /csp.php");
```

Comment:
I think listing the policies in an array and imploding that is more readable and easily extendable:

```suggestion
/** Content-Security-Policy headers */

$policies = [
    'default-src \'self\'',
    'style-src \'self\' \'unsafe-inline\' https://fonts.googleapis.com https://cdnjs.cloudflare.com https://*.bootstrapcdn.com',
    'script-src \'self\' \'unsafe-inline\' \'unsafe-eval\' https://cdnjs.cloudflare.com',
    'img-src \'self\' data: https://cdnjs.cloudflare.com',
    'font-src \'self\' data: https://*.bootstrapcdn.com https://fonts.gstatic.com',
    'form-action \'self\'',
    'upgrade-insecure-requests',
    'block-all-mixed-content',
    'report-uri /csp.php',
];

header('Content-Security-Policy-Report-Only: '.implode('; ', $policies));
```